unity3d/Assets/Swrve/SwrveSDK/SwrveComponent.cs

```diff
@@ -64,10 +64,10 @@ public static SwrveComponent Instance
     }
 
     /// <summary>
-    /// Default constructor. Will be called by Unity when
-    /// placing this script in your scene.
+    /// Reset is invoked when the component is added to the Game Object
+    /// and is always called on the main thread.
     /// </summary>
-    public SwrveComponent ()
+    public void Reset ()
```

Comment:
I believe Reset is only an Editor callback. It might be better to use "OnEnable".

Could you let us know how we can reproduce your problem? We don't have that issue internally.
What exceptions is it throwing?

Thanks!